dev-lang/spidermonkey/spidermonkey-1.8.5.recipe

```diff
@@ -41,13 +42,14 @@ BUILD_PREREQUIRES="
 	cmd:xargs
 	cmd:pkg_config$secondaryArchSuffix
 	cmd:sed
+	cmd:zip
 	"
 
 BUILD()
 {
 	cd js/src
 	runConfigure --omit-dirs "libDir docDir dataRootDir" ./configure \
```

Comment:
having libDir used correctly is required for secondary arch support, currently INSTALL is trying to find something in lib/x86 which wasn't installed...